crates/client/assets/schema.sdl

```diff
@@ -871,6 +871,12 @@ type Query {
 		"""
 		id: BlobId!
 	): Blob
+	blobExists(
```

Comment:
this shouldn't require any changes to the schema

you can just add a method to fuel-core-client that does the same kind of existence query as the ts-sdk

Reply:
<img width="1049" alt="Screenshot 2024-10-05 at 11 01 20 AM" src="https://github.com/user-attachments/assets/f51ec682-9868-44f9-b1cf-59602dc99376">

When you query a blob by id and ask for the id back, it will return null if it doesn't exist